.github/workflows/test-terra-jupyter-aou.yml

```diff
@@ -0,0 +1,112 @@
+name: Test terra-jupyter-aou
+# Perform smoke tests on the terra-jupyter-aou Docker image to have some amount of confidence that
+# Python package versions are compatible.
+#
+# To configure the minimal auth needed for these tests to be able to read public data from Google Cloud Platform:
+# Step 1: Create a service account per these instructions:
+#         https://github.com/google-github-actions/setup-gcloud/blob/master/setup-gcloud/README.md
+# Step 2: Give the service account the following permissions within the project: BigQuery User
+# Step 3: Store its key and project id as GitHub repository secrets GCP_SA_KEY and GCP_PROJECT_ID.
```

Comment:
Nice comments. :) Makes sense, thinking we can use the `broad-dsp-gcr-public` project for this.

Reply:
> Nice comments. :) Makes sense, thinking we can use the `broad-dsp-gcr-public` project for this.

@Qi77Qi here is the project @rtitle recommended.